### ISSUE TITLE
New submissions for Tue, 15 Mar 22

### ISSUE THREAD

**Issue by imjunwei** [imjunwei/ocr-daily-paper#153]
### Kernel Proposal Network for Arbitrary Shape Text Detection
 - **Authors:** Shi-Xue Zhang, Xiaobin Zhu, Jie-Bo Hou, Chun Yang, Xu-Cheng Yin
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.06410
 - **Pdf link:** https://arxiv.org/pdf/2203.06410
 - **Abstract**
 Segmentation-based methods have achieved great success for arbitrary shape text detection. However, separating neighboring text instances is still one of the most challenging problems due to the complexity of texts in scene images. In this paper, we propose an innovative Kernel Proposal Network (dubbed KPN) for arbitrary shape text detection. The proposed KPN can separate neighboring text instances by classifying different texts into instance-independent feature maps, meanwhile avoiding the complex aggregation process existing in segmentation-based arbitrary shape text detection methods. To be concrete, our KPN will predict a Gaussian center map for each text image, which will be used to extract a series of candidate kernel proposals (i.e., dynamic convolution kernel) from the embedding feature maps according to their corresponding keypoint positions. To enforce the independence between kernel proposals, we propose a novel orthogonal learning loss (OLL) via orthogonal constraints. Specifically, our kernel proposals contain important self-information learned by network and location information by position embedding. Finally, kernel proposals will individually convolve all embedding feature maps for generating individual embedded maps of text instances. In this way, our KPN can effectively separate neighboring text instances and improve the robustness against unclear boundaries. To our knowledge, our work is the first to introduce the dynamic convolution kernel strategy to efficiently and effectively tackle the adhesion problem of neighboring text instances in text detection. Experimental results on challenging datasets verify the impressive performance and efficiency of our method. The code and model are available at https://github.com/GXYM/KPN.
### Training Protocol Matters: Towards Accurate Scene Text Recognition via  Training Protocol Searching
 - **Authors:** Xiaojie Chu, Yongtao Wang, Chunhua Shen, Jingdong Chen, Wei Chu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.06696
 - **Pdf link:** https://arxiv.org/pdf/2203.06696
 - **Abstract**
 The development of scene text recognition (STR) in the era of deep learning has been mainly focused on novel architectures of STR models. However, training protocol (i.e., settings of the hyper-parameters involved in the training of STR models), which plays an equally important role in successfully training a good STR model, is under-explored for scene text recognition. In this work, we attempt to improve the accuracy of existing STR models by searching for optimal training protocol. Specifically, we develop a training protocol search algorithm, based on a newly designed search space and an efficient search algorithm using evolutionary optimization and proxy tasks. Experimental results show that our searched training protocol can improve the recognition accuracy of mainstream STR models by 2.7%~3.9%. In particular, with the searched training protocol, TRBA-Net achieves 2.1% higher accuracy than the state-of-the-art STR model (i.e., EFIFSTR), while the inference speed is 2.3x and 3.7x faster on CPU and GPU respectively. Extensive experiments are conducted to demonstrate the effectiveness of the proposed method and the generalization ability of the training protocol found by our search method.
### Mathematically Quantifying Gerrymandering and the Non-responsiveness of  the 2021 Georgia Congressional Districting Plan
 - **Authors:** Zhanzhan Zhao, Cyrus Hettle, Swati Gupta, Jonathan Mattingly, Dana Randall, Gregory Herschlag
 - **Subjects:** Computers and Society (cs.CY); Computation (stat.CO)
 - **Arxiv link:** https://arxiv.org/abs/2203.06552
 - **Pdf link:** https://arxiv.org/pdf/2203.06552
 - **Abstract**
 While gerrymandering has been widely suspected in Georgia for years, it has been difficult to quantify. Using historic voting data, we compare the Georgia congressional redistricting plan enacted in 2021 with a large collection of randomly generated non-partisan maps. We find that the 2021 plan will likely be highly non-responsive to changing opinions of the electorate. The enacted map is structured so that it will reliably elect nine Republicans and five Democrats over a wide range of studied voting patterns. In contrast, for the non-partisan maps in our ensemble, only 0.12% of the plans behave similarly to the enacted plan over the seventeen elections considered between 2016-2020. We find the major cause of the non-responsive nature of the 2021 map is that it polarizes voters across the more competitive districts. Specifically, there are five districts that would be significantly more competitive under maps with our non-partisan considerations but have been shifted to become more Republican; similarly, there are three districts that have more Democratic voters than is typical. Utilizing a combination of our ensemble statistics and spatial analysis, we find evidence that the heavily Democratic 4th, 5th and 13th Congressional Districts and the heavily Republican 9th District contain an unrepresentatively large number of Democrats. The consolidation or packing of Democratic voters in the 4th, 5th, and 13th Districts creates significant numbers of wasted votes and dilutes their voting power. Furthermore, based on comparisons with the previously enacted 2011 map and the ensemble of plans, we find that a significant number of Democratic voters have been added to the solidly Republican 9th District, replacing Republican voters who have been moved to the 6th and 10th Districts. This redrawing substantially weakens, or cracks, the potential influence of Democratic votes in these districts.
### XYLayoutLM: Towards Layout-Aware Multimodal Networks For Visually-Rich  Document Understanding
 - **Authors:** Zhangxuan Gu, Changhua Meng, Ke Wang, Jun Lan, Weiqiang Wang, Ming Gu, Liqing Zhang
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2203.06947
 - **Pdf link:** https://arxiv.org/pdf/2203.06947
 - **Abstract**
 Recently, various multimodal networks for Visually-Rich Document Understanding(VRDU) have been proposed, showing the promotion of transformers by integrating visual and layout information with the text embeddings. However, most existing approaches utilize the position embeddings to incorporate the sequence information, neglecting the noisy improper reading order obtained by OCR tools. In this paper, we propose a robust layout-aware multimodal network named XYLayoutLM to capture and leverage rich layout information from proper reading orders produced by our Augmented XY Cut. Moreover, a Dilated Conditional Position Encoding module is proposed to deal with the input sequence of variable lengths, and it additionally extracts local layout information from both textual and visual modalities while generating position embeddings. Experiment results show that our XYLayoutLM achieves competitive results on document understanding tasks.
### CAR: Class-aware Regularizations for Semantic Segmentation
 - **Authors:** Ye Huang, Di Kang, Liang Chen, Xuefei Zhe, Wenjing Jia, Xiangjian He, Linchao Bao
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.07160
 - **Pdf link:** https://arxiv.org/pdf/2203.07160
 - **Abstract**
 Recent segmentation methods, such as OCR and CPNet, utilizing "class level" information in addition to pixel features, have achieved notable success for boosting the accuracy of existing network modules. However, the extracted class-level information was simply concatenated to pixel features, without explicitly being exploited for better pixel representation learning. Moreover, these approaches learn soft class centers based on coarse mask prediction, which is prone to error accumulation. In this paper, aiming to use class level information more effectively, we propose a universal Class-Aware Regularization (CAR) approach to optimize the intra-class variance and inter-class distance during feature learning, motivated by the fact that humans can recognize an object by itself no matter which other objects it appears with. Three novel loss functions are proposed. The first loss function encourages more compact class representations within each class, the second directly maximizes the distance between different class centers, and the third further pushes the distance between inter-class centers and pixels. Furthermore, the class center in our approach is directly generated from ground truth instead of from the error-prone coarse prediction. Our method can be easily applied to most existing segmentation models during training, including OCR and CPNet, and can largely improve their accuracy at no additional inference overhead. Extensive experiments and ablation studies conducted on multiple benchmark datasets demonstrate that the proposed CAR can boost the accuracy of all baseline models by up to 2.23% mIOU with superior generalization ability. The complete code is available at https://github.com/edwardyehuang/CAR.
## Keyword: Handwriting
There is no result 
### Training Protocol Matters: Towards Accurate Scene Text Recognition via  Training Protocol Searching
 - **Authors:** Xiaojie Chu, Yongtao Wang, Chunhua Shen, Jingdong Chen, Wei Chu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2203.06696
 - **Pdf link:** https://arxiv.org/pdf/2203.06696
 - **Abstract**
 The development of scene text recognition (STR) in the era of deep learning has been mainly focused on novel architectures of STR models. However, training protocol (i.e., settings of the hyper-parameters involved in the training of STR models), which plays an equally important role in successfully training a good STR model, is under-explored for scene text recognition. In this work, we attempt to improve the accuracy of existing STR models by searching for optimal training protocol. Specifically, we develop a training protocol search algorithm, based on a newly designed search space and an efficient search algorithm using evolutionary optimization and proxy tasks. Experimental results show that our searched training protocol can improve the recognition accuracy of mainstream STR models by 2.7%~3.9%. In particular, with the searched training protocol, TRBA-Net achieves 2.1% higher accuracy than the state-of-the-art STR model (i.e., EFIFSTR), while the inference speed is 2.3x and 3.7x faster on CPU and GPU respectively. Extensive experiments are conducted to demonstrate the effectiveness of the proposed method and the generalization ability of the training protocol found by our search method.
## Keyword: Text Segmentation
There is no result 
## Keyword: TextSpotter
There is no result 
## Keyword: Text Spotting
There is no result
